modules/ismp/core/src/consensus.rs

```diff
@@ -122,6 +122,53 @@ pub struct StateMachineHeight {
 	pub height: u64,
 }
 
+/// 6s slot or 12s slot duration based on current parablock progression by the relaychain mechanism
+#[derive(
+	Debug,
+	Clone,
+	Copy,
+	Encode,
+	Decode,
+	scale_info::TypeInfo,
+	MaxEncodedLen,
+	PartialEq,
+	Eq,
+	Hash,
+	Ord,
+	PartialOrd,
+	serde::Deserialize,
+	serde::Serialize,
+)]
+pub enum ParaSlotDuration {
+	/// synchronous backed
+	Sync,
+	/// asynchronous backed
+	Async,
+}
+/// Data provided when registering a parachain to be tracked by hyperbridge consensus client
+#[derive(
+	Debug,
+	Clone,
+	Copy,
+	Encode,
+	Decode,
+	scale_info::TypeInfo,
+	MaxEncodedLen,
+	PartialEq,
+	Eq,
+	Hash,
+	Ord,
+	PartialOrd,
+	serde::Deserialize,
+	serde::Serialize,
+)]
+pub struct ParachainData {
+	/// parachain id
+	pub id: u32,
+	/// parachain slot duration type
+	pub slot_duration_type: ParaSlotDuration,
+}
+
```

Comment:
This shouldn't be here, move it to ismp-parachain. Also just track the slot duration directly rather than the slot duration type